packages/terra-list/src/ListSectionHeader.module.scss

```
@@ -0,0 +1,79 @@
+@import '~terra-mixins';
+
+// Section Header
+// ========================================================
+:local {
+  .section-header {
+    background-color: var(--terra-list-v3-section-header-background-color, #e8e9ea);
+    border-bottom: var(--terra-list-v3-section-header-border-bottom, 1px solid #c8cacb);
+    padding-bottom: var(--terra-list-v3-section-header-padding-bottom, 0.5rem);
+    padding-left: var(--terra-list-v3-section-header-padding-left, 0.7142857143rem);
+    padding-top: var(--terra-list-v3-section-header-padding-top, 0.5rem);
+    width: 100%;
+  }
+
+  .title {
+    color: var(--terra-list-v3-section-header-title-color, #1c1f21);
+    font-size: var(--terra-list-v3-section-header-title-font-size, 1rem);
+    font-weight: var(--terra-list-v3-section-header-title-font-weight, bold);
+    line-height: var(--terra-list-v3-section-header-title-line-height, 1.1428571429);
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    word-break: break-all;
```

Let's change this to the following:

```suggestion
    overflow-wrap: break-word; /* Modern browsersrs */
    word-wrap: break-word; /* For IE 10 and IE 11 */
```

---

Updated.